src/parameters.toml

```diff
@@ -38,6 +37,93 @@ value = 0.25
 type = "float"
 description = "Factor of relative contribution or Rgrowth (-)"
 
+
+# Farquhar Parameters
+
+[CO2_compensation_point_25c]
+value = 4.275e-5
+type = "float"
+description = "Γstar at 25 °C (mol/mol)"
+
+[CO2_michaelis_menten]
+value = 4.049e-4
+type = "float"
+description = "Michaelis-Menten parameter for CO2 at 25 °C (mol/mol)"
+
+[O2_michaelis_menten]
+value = 0.2874
+type = "float"
+description = "Michaelis-Menten parameter for O2 at 25 °C (mol/mol)"
+
+[CO2_activation_energy]
+value = 79430
+type = "float"
+description = "Energy of activation for CO2 (J/mol)"
+
+[O2_activation_energy]
+value = 36380
+type = "float"
+description = "Energy of activation for oxygen (J/mol)"
+
+[Vcmax_activation_energy]
+value = 58520
+type = "float"
+description = "Energy of activation for Vcmax (J/mol)"
+
+["Γstar_activation_energy"]
+value = 58520
+type = "float"
+description = "Energy of activation for Γstar (J/mol)"
+
+[Jmax_activation_energy]
+value = 43540
+type = "float"
+description = "Energy of activation for Jmax (J/mol)"
+
+[Rd_activation_energy]
+value = 46390
+type = "float"
+description = "Energy of activation for Rd (J/mol)"
+
+[kelvin_25C]
+value = 298.15
+type = "float"
+description = "Reference temperature equal to 25 degrees Celsius (K)"
+
+[intercellular_CO2_concentration]
+value = 0.209
+type = "float"
+description = "Intercelluar O2 concentration (mol/mol); taken to be constant"
+
+[photosystem_II_quantum_yield]
+value = 0.6
+type = "float"
+description = "Quantum yield of photosystem II (Bernacchi, 2003; unitless)"
+
+[Farquhar_curvature_parameter]
+value = 0.6
```

Comment:
I believe we are using 0.9 for this

Reply:
We can cite (von Caemmerer 2000,2013; Bernacchi et al.2003,2013; von Caemmerer et al. 2009).